.github/workflows/test.yml

```
@@ -97,7 +97,7 @@ jobs:
           # but it's useful to run them with tox too,
           # to ensure the tox env works as expected
         - name: Formatting with Black + isort and code style with flake8
-          python: '3.7'
+          python: '3.10'
```

Any reason not to use 3.11 here?

```suggestion
          python: '3.11'
```

---

It felt slightly safer to use 3.10 😆. I still feel that 3.11 is new :P.